### PR TITLE
ngraph python sample

### PR DESCRIPTION
**Pull request by mryzhov**
This sample demonstrates how to execute an inference using ngraph::Function to create a network
- added sample
- added readme
- added lenet weights

### CODE REVIEW

File: inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+    return images
+
+
+def read_image(image_path: np):
+    # try to read image in usual image formats
+    image = cv2.imread(image_path, cv2.IMREAD_UNCHANGED)
+
+    # try to open image as ubyte
+    if image is None:
+        image_file = open(image_path, 'rb')
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py:69]: maybe use `with` context manager

File: inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+
+    # Processing results
+    log.info("Processing output blob")
+    res = res[out_blob]
+    log.info("Top {} results: ".format(args.number_top))
+    if args.labels:
+        with open(args.labels, 'r') as f:
+            labels_map = [x.split(sep=' ', maxsplit=1)[-1].strip() for x in f]
+    else:
+        labels_map = None
```
Comment (generalova-kate) [inline inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py:247]: you can remove else and set labels_map = None before if statement

File: inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py (hunk 1 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py]: thanks, done

File: inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py (hunk 2 above)
Comment (mryzhov) [reply inference-engine/ie_bridges/python/sample/ngraph_function_creation_sample/ngraph_function_creation_sample.py]: not an issue, but looks better, fixed

**Review by generalova-kate — APPROVED** [review]
Looks good

**Pull request merged**